sdk/containerregistry/azure-containers-containerregistry-perf/pom.xml

```diff
@@ -18,6 +18,7 @@
   <packaging>jar</packaging>
 
   <properties>
+    <surefireXmx>1024m</surefireXmx>
```

Comment:
Any reason we're using a lower memory limit in performance testing? If we want to test out memory constraints look at how Storage does it with an additional pipeline run during live testing:

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/tests-template.yml#L75

We could look into standardizing this if we believe more libraries should be doing this form of validation/testing

Reply:
I tried it for local testing and did not  remove, happy to do it in a same way as storage